### ISSUE TITLE
Method 2 invalid for species - but the "species" in question has been superseded

### ISSUE THREAD

**Issue by atcooper1** [aodn/nrmn-application#658]
The superseding checks need to happen before some of the validation checks. Currently getting "Method 2 invalid for species" warning errors but the species listed in the errors have been correctly superseded by another species.
E.g. in Canada Bamfield _Peltodoris nobilis_ has been superseded by _Montereina nobilis_, but the invalid warning error is currently 
for _Peltodoris nobilis_ - the superseding needs to be applied before this validation check.
At what stage are supersedings applied?

**Comment by bpasquer** [comment]
Superseding are not applied in the NRMN ingest tool as it was in the RLS data assistant. 
Superseded species are validated as any other species. 

In the above case the issue is that taxonomy of superseded species used to be deleted in the RLS DB ( see https://github.com/aodn/nrmn-application/issues/726#issuecomment-995401952). Because these species have missing information, NRMN taxonomic rules ( e.g. methods -species) do not applied to them,  nor do endpoint taxonomic rules which as a consequence means that they are excluded from the endpoints  . This issue impact ~300 species.
The resolution of this issue will be followed up in https://github.com/aodn/nrmn-application/issues/758
